src/app/components/normal-component/pick-game/pick-game.component.ts

```diff
@@ -39,6 +40,8 @@ class GameDescription {
 
     public static readonly COERCEO: Localized = () => $localize`Get rid of all of your opponent's pieces on a board that shrinks little by little!`;
 
+    public static readonly CONSPIRATEURS: Localized = () => $localize`Hide all of your pieces before your opponent does, or risk to be discovered!`;
```

Comment:
So, keeping the french name in english as well :p ?

Reply:
oui, comme wikipédia !